### ISSUE TITLE
PYZO_PROCESS_EVENTS_WHILE_DEBUGGING has no effect

### ISSUE THREAD

**Issue by divenex** [pyzo/pyzo#535]
Using Pyzo 4.5.1 with Python 3.6.5 under Windows 10, the keyword `PYZO_PROCESS_EVENTS_WHILE_DEBUGGING` has no effect. The problem remains both when using the Windows installation executable, or when installing Pyzo using `pip`.

Specifically, when setting this variable to 0 or to 1 in my shell configuration, in both cases Matplotlib plots do not appear while debugging: a window shows up but is empty and "Not Responding". 

Note: the figures do appear during debugging, after typing multiple `db events` or using `plt.pause(1)`. But this alternative is not very convenient. (FYI: I had this feature working properly with the old IEP)

**Comment by divenex** [comment]
Yes, everything else works beautifully. Here is the shell starting message

Python 3.6.5 |Anaconda, Inc.| (default, Mar 29 2018, 13:32:41) on Windows (64 bits).
This is the Pyzo interpreter with integrated event loop for PYQT5.
Type 'help' for help, type '?' for a list of *magic* commands.

**Comment by almarklein** [comment]
Before diving deeper, how do you set the environment variable? Do you use the shell config for that. If so, you need to restart the kernel for it to take effect. If you set the environment variable in your OS, then you'd need to restart Pyzo as a whole. To be sure, check `os.getenv('PYZO_PROCESS_EVENTS_WHILE_DEBUGGING')` in the shell.

**Comment by divenex** [comment]
Hi Almar, thank you for the very useful feedback.

<p>I had set the environment variable from the Pyzo shell config as follows:

<p>![pyzo](https://user-images.githubusercontent.com/9269490/39294563-f93c7994-4933-11e8-9f72-91dd644d00ae.png)

<p>I tried the command you suggested and noticed that the environment variable was actually empty. 

<p>So, following your comment, I set the Pyzo environment variable by hand in the Windows 10 OS, and all worked fine, and the command `os.getenv('PYZO_PROCESS_EVENTS_WHILE_DEBUGGING')` properly returned `1`.

<p>I finally realized that the problem lies in the extra whitespace character I had before the `=` sign! Removing the space in the Pyzo shell config, fixed the problem. 

<p>Is this behaviour with whitespace characters intended?
